app/models/like.rb

```diff
@@ -1,2 +1,9 @@
 class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def count_updater(post_id)
+    post = Post.find_by(id: post_id)
+    post.likes_counter = likes.where(post_id:).count
+  end
```

Comment:
The same thing occurs here. Please remember to use the `after_save` callback to call this method once a `like` is created and saved.